File: solutions/csharp/0700-search-in-a-binary-search-tree.cs
public class Solution {
    public TreeNode SearchBST(TreeNode root, int val) {
        if (root == null) {
            return null;
        }
        if (root.val == val) {
            return root;
        }
        if (root.val < val) {
            return SearchBST(root.right, val);
        }
        return SearchBST(root.left, val);
    }
}

File: solutions/python/0700-search-in-a-binary-search-tree.py
class Solution:
    def searchBST(self, root: Optional[TreeNode], val: int) -> Optional[TreeNode]:
        if root is None: 
            return None 
        if root.val == val: 
            return root 
        elif root.val < val: 
            return self.searchBST(root.right, val) 
        else:  
            return self.searchBST(root.left, val)    
